### PR TITLE
Added a task to the PR template to add instructions to the release testing doc

### LINKED ISSUE #2735: Add a task to the PR template to add testing instructions to the release testing doc

### PR DESCRIPTION
**Pull request by aprea**
Fixes #2735

#### Changes proposed in this Pull Request

As per the PR title.

I have introduced this task as a gentle reminder that new features/bug fixes that require GlobalStep testing should be added to the release testing document.

#### Testing instructions

* View here 👉 https://github.com/Automattic/woocommerce-payments/blob/new/2735-add-testing-instructions-reminder/.github/PULL_REQUEST_TEMPLATE.md

-------------------

- [x] Added changelog entry (or does not apply)
- [x] Covered with tests (or have a good reason not to test in description ☝️)
- [x] Tested on mobile (or does not apply)

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -38,3 +38,12 @@ Add as many details as possible to help others reproduce the issue and test the
...
 - [ ] Tested on mobile (or does not apply)
+
+**Post merge**
+
+<!--
+Make sure you edit the page for the current release when adding testing instructions.
+We often create a blank page ahead of time for the next release.
+-->
+
+- [ ] Added testing instructions to the [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) (or does not apply)
```
Comment (naman03malhotra) [inline .github/PULL_REQUEST_TEMPLATE.md:49]: I have a suggestion that can make _what's new post_ writing easier. If the change impacts the UI somehow, then the author can add a screenshot for the same in the PR description. That can save release lead's time to try out that feature and then click the screenshot (sometimes it takes more time due to lack of context for that feature). What do you think?

**Review by naman03malhotra — COMMENTED** [review]
Nice addition Chris! 🎉 
LGTM, left a suggestion for you to consider.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (aprea) [reply .github/PULL_REQUEST_TEMPLATE.md:49]: It's a good idea but I might leave it for another PR to keep this one single focus.

Also, we already have the following comment in the template:
```
<!--
Images or gifs: Include before and after screenshots or gifs/videos when it makes sense.
-->
```

I'd hope most people are including screenshots when appropriate.

**Review by htdat — APPROVED** [review]

**Comment by htdat** [comment]
LGTM!

**Pull request merged**